backend/FwLite/MiniLcm.Tests/BasicApiTestsBase.cs

```diff
@@ -362,8 +362,9 @@ public async Task CreateSense_WillCreateWithExistingDomains()
     public async Task CreateSense_WontCreateMissingPartOfSpeech()
```

Comment:
I actually think that the design this test it validating is bad. If you try to reference a part of speech which doesn't exist then it should fail with an error. @rmunn would you make that change? both to the test and to the API, this would be a precondition in the API which throws if the part of speech Id does not exist.

Reply:
I spent way too long on this, but I finally got it working in commit 97bf7358. Please let me know if this is what you were thinking of.

Reply:
that looks great!